.github/workflows/publish-docker.yml

```diff
@@ -18,6 +18,9 @@ jobs:
         project:
           - bridge-node
           - ethereum-poa-relay
+        healthcheck:
```

Comment:
I think this will create all possible combinations. We should rather change `HEALTH` based on `matrix.project`